janusgraph-solr/pom.xml

```diff
@@ -56,19 +54,27 @@
             <groupId>org.apache.solr</groupId>
             <artifactId>solr-solrj</artifactId>
             <version>${solr.version}</version>
+            <exclusions>
+                <!-- Use jcl-over-slf4j from gremlin-core -->
+                <exclusion>
+                    <groupId>org.slf4j</groupId>
+                    <artifactId>jcl-over-slf4j</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.apache.solr</groupId>
```

Comment:
@wojciechwojcik Okay not sure what I was looking at earlier but when I try to remove solr-core from janusgraph-solr there's a build error. It might be possible to update to replace solr-core with other (e.g. lucene) dependencies but I think the underlying issue in #310 (e.g. failures when including both janusgraph-lucene and janusgraph-solr) should be resolved just by lining up Lucene versions across the modules as is done in this branch.